src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
Just to make sure.  Before the new version, we had 2 previous versions:
<img width="299" alt="screen shot 2017-06-18 at 12 45 58 am" src="https://user-images.githubusercontent.com/1297448/27256504-93b70504-53bf-11e7-8551-6d65519cc3d2.png">
which branches are they match currently?